src/otx/core/utils/tile_merge.py

```diff
@@ -333,7 +356,12 @@ def merge(
                 feature_vectors,
                 strict=True,
             ):
-                keep_indices = tile_masks.to_sparse().sum((1, 2)).to_dense() > 0
+                # NOTE: RuntimeError: nonzero is not supported for tensors with more than INT_MAX elements,
+                # See https://github.com/pytorch/pytorch/issues/51871
```

Comment:
Looks like the issue was resolved, is it possible to track down torch version to mention here? Is it 2.6 already?